tomviz/CentralWidget.ui

```diff
@@ -96,15 +68,15 @@
  </widget>
  <customwidgets>
   <customwidget>
-   <class>pqTabbedMultiViewWidget</class>
+   <class>QVTKWidget</class>
    <extends>QWidget</extends>
-   <header>pqTabbedMultiViewWidget.h</header>
+   <header location="global">qvtkwidget.h</header>
```

Comment:
Ah, thanks for catching. Designer must have done this. Actually, I don't know why the pqTabbedMultiViewWidget was changed at all. Let me revisit, but thanks for pointing this out.